cdqa/utils/converters.py

```diff
@@ -135,12 +157,54 @@ def pdf_converter(directory_path):
             df.loc[i] = [pdf, None]
             raw = parser.from_file(os.path.join(directory_path, pdf))
             s = raw["content"]
-            paragraphs = re.split(u"\n(?=\u2028|[A-Z-0-9])", s)
+            paragraphs = re.split("\n(?=\u2028|[A-Z-0-9])", s)
             list_par = []
+            temp_para = (
+                ""
+            )  # varible that stores paras with length<200 (considered as a line)
             for p in paragraphs:
-                if len(p) >= 200:
-                    list_par.append(p.replace("\n", ""))
-                df.loc[i, "paragraphs"] = list_par
+                if not p.isspace():  # checking if para is not only spaces
+                    if include_line_breaks:
+                        """
```

Comment:
Usually we avoid using `""""` inside the function / method, we use it only for the description of the method / function / class. It's better to use `#`.

The reason for this is that if a user wants to check the docstring of the function (with `Shift+tab` inside a notebook or using `?` in an ipython console) these docstrings will appear to them. We want to avoid that and make it available only for people who will be reading the code.

Reply:
Check!